test/Benchmarks/Metrics/ExemplarBenchmarks.cs

```diff
@@ -23,19 +23,21 @@
 
 /*
 // * Summary *
-BenchmarkDotNet=v0.13.3, OS=Windows 11 (10.0.22621.1265)
-11th Gen Intel Core i7-1185G7 3.00GHz, 1 CPU, 8 logical and 4 physical cores
+BenchmarkDotNet=v0.13.3, OS=Windows 10 (10.0.19045.2604)
+Intel Core i7-4790 CPU 3.60GHz (Haswell), 1 CPU, 8 logical and 4 physical cores
 .NET SDK=7.0.103
   [Host]     : .NET 7.0.3 (7.0.323.6910), X64 RyuJIT AVX2
   DefaultJob : .NET 7.0.3 (7.0.323.6910), X64 RyuJIT AVX2
 
 
-|                    Method | EnableExemplar |     Mean |   Error |  StdDev | Allocated |
-|-------------------------- |--------------- |---------:|--------:|--------:|----------:|
-|   HistogramNoTagReduction |          False | 256.5 ns | 4.84 ns | 4.53 ns |         - |
-| HistogramWithTagReduction |          False | 246.6 ns | 4.90 ns | 4.81 ns |         - |
-|   HistogramNoTagReduction |           True | 286.4 ns | 5.30 ns | 7.25 ns |         - |
-| HistogramWithTagReduction |           True | 293.6 ns | 5.77 ns | 7.09 ns |         - |
+|                    Method | ExemplarFilter |     Mean |   Error |  StdDev |
+|-------------------------- |--------------- |---------:|--------:|--------:|
+|   HistogramNoTagReduction |      AlwaysOff | 380.7 ns | 5.92 ns | 5.53 ns |
+| HistogramWithTagReduction |      AlwaysOff | 356.5 ns | 3.33 ns | 2.95 ns |
+|   HistogramNoTagReduction |       AlwaysOn | 412.3 ns | 2.11 ns | 1.64 ns |
+| HistogramWithTagReduction |       AlwaysOn | 461.0 ns | 4.65 ns | 4.35 ns |
```

Comment:
Add another params for number of dimensions for benchmarks that do tag reduction?

Reply:
it'll clutter a lot.. as I am adding more and more to this now. Will find a balanced approach before finalizing.. Too much makes this cluttered. Too less makes it hard to convey which ones end-users should focus on..

Reply:
Yeah we could have them in a different benchmarks class.